diff --git a/PKGBUILD b/PKGBUILD
index da21044..4903d05 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -326,6 +326,7 @@ package_kodi-bin-devel() {
     'libxslt' 'lirc' 'mariadb-libs' 'python2' 'smbclient' 'taglib'
     'tinyxml' "$pkgbase"
   )
+  provides=("kodi-bin=${pkgver}" 'kodi-x11')
 
   cd kodi-build-x11
   install -Dm755 kodi-x11 "$pkgdir/usr/lib/kodi/kodi-x11"
@@ -337,7 +338,7 @@ package_kodi-bin-devel() {
 
 package_kodi-wayland-devel() {
   pkgdesc="wayland kodi binary"
-  provides=("kodi-bin=${pkgver}")
+  provides=("kodi-bin=${pkgver}" "kodi-wayland")
   conflicts=('kodi-wayland')
   depends=(
     'bluez-libs' 'curl' 'lcms2' 'libass' 'libbluray' 'libcdio' 'libcec'
